lib/rust/ensogl/examples/mouse-events/src/lib.rs

```diff
@@ -61,17 +46,27 @@ mod shape {
 struct Model {
     app:            Application,
     display_object: display::object::Instance,
-    shape:          shape::View,
+    shape:          Rectangle,
+    cover:          Rectangle,
 }
 
 impl Model {
     fn new(app: &Application) -> Self {
         let app = app.clone_ref();
         let display_object = display::object::Instance::new();
-        let shape = shape::View::new();
-        shape.set_size(Vector2::new(100.0, 100.0));
+        let shape: Rectangle = default();
+        shape.set_size(Vector2::new(300.0, 300.0));
```

Comment:
Instead of `Vector2::new(100.0, 100.0)`, you can simply do `Vector2(100.0, 100.0)`.